x-pack/platform/plugins/shared/fleet/common/mocks.ts

```diff
@@ -5,9 +5,13 @@
  * 2.0.
  */
 
+import { type DeepPartial } from 'utility-types';
```

Comment:
I'm not sure what the difference is, but I saw that this is also available from `@kbn/utility-types`

Reply:
Seems we use the `@kbn` one more, so I'll source from that.